UnitsNet/UnitDefinitions/FoulingFactor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "FoulingFactor",
```

Comment:
Based on the units listed below, I found this page:
http://www.ambrsoft.com/UnitsConverterCalc/ThermalResistance.htm

It seems to me the units that contain a time unit, are `ThermalResistance` units, while time-less units are `ThermalResistivity` ? Would that be correct?